site/src/pages/HealthPage/WorkspaceProxyPage.tsx

```diff
@@ -138,14 +142,17 @@ export const WorkspaceProxyPage = () => {
                   color: theme.palette.text.secondary,
                 }}
               >
-                {warnings.length > 0 ? (
+                {warnings.length === 0 && errors.length === 0 ? (
+                  <span>OK</span>
+                ) : (
                   <div css={{ display: "flex", flexDirection: "column" }}>
+                    {errors.map((error, i) => (
+                      <span key={i}>{error}</span>
+                    ))}
```

Comment:
I might be wrong but this form should work too?

```tsx
{[...errors, ...warnings].map((message, i) => (
      <span key={i}>{message}</span>
    ))}
```

Reply:
I like this.